tests/lib/lte_lc/CMakeLists.txt

```diff
@@ -18,19 +18,48 @@ cmock_handle(${ZEPHYR_NRFXLIB_MODULE_DIR}/nrf_modem/include/nrf_modem_at.h
 FILE(GLOB app_sources src/*.c)
 target_sources(app PRIVATE ${app_sources})
 
+FILE(GLOB lib_module_sources ${ZEPHYR_NRF_MODULE_DIR}/lib/lte_link_control/modules/*.c)
+FILE(GLOB lib_common_sources ${ZEPHYR_NRF_MODULE_DIR}/lib/lte_link_control/common/*.c)
```

Comment:
why are library files includes here instead of being linker from the Zephyr library ?

Also the use of glob for selecting source files is generally considered a bad practice in CMake, but that's another story.

Reply:
What do you mean with 

> instead of being linker from the Zephyr library ?

I'm not aware of any other way to achieve this same result. Originally, this test includes `lte_lc_helpers.c` in the same way (but with `target_sources(app PRIVATE ${ZEPHYR_NRF_MODULE_DIR}/lib/lte_link_control/lte_lc_helpers.c`). If there is a better way to achieve this I'm open to hear any suggestions.

Reply:
tests/lib/lte_lc have been removed so resolving this comment.

Reply:
> Regarding the use of GLOB, this is what I learned from other CMakeList files

yes, we allow it for the sample's / test's top-level CMakeLists.txt file for the files directly residing in the sample's / test's (sub-)folders.

But in this particular case, the glob was used to pick up files from: `${ZEPHYR_NRF_MODULE_DIR}/lib/lte_link_control/...` and hence the reason for my comment.